src/test/java/hudson/plugins/warnings/parser/JavacParserTest.java

```diff
@@ -91,6 +91,18 @@ public void parseArrayInDeprecatedMethod() throws IOException {
                 WARNING_TYPE, "Deprecation", Priority.NORMAL);
     }
 
+    /**
+     * Parses parallel pipeline output based on 'javac.txt'
+     *
+     * @throws IOException Signals that an I/O exception has occurred.
+     */
+    @Test
+    public void parseParallelPipelineOutput() throws IOException {
+        Collection<FileAnnotation> warnings = parse("javac-parallel-pipeline.txt");
+
+        assertEquals(WRONG_NUMBER_OF_WARNINGS_DETECTED, 2, warnings.size());
+    }

```

Comment:
Add checks for the warnings properties: `checkWarning(...)`

Then we are sure, that the regexp works correctly.